client/src/store/new-request-module.ts

```diff
@@ -496,60 +547,64 @@ export class NewRequestModule extends VuexModule {
   }
   requestActions: RequestActionsI[] = [
     {
-      text: 'Start a New',
+      text: 'Start a new',
       shortDesc: 'New Request',
       value: 'NEW',
-      blurb: `Start a new business in BC. This applies to starting fresh from here or having a business in another 
-              province or country that you want to operate in BC as well.`
+      blurb: `Create a new business in British Columbia or register a business you formed in another province or 
+              territory, country or federal jurisdiction so that you may also conduct business here in BC.`
     },
     {
-      text: 'Move your Business to BC',
+      text: 'Move (continue) into BC a',
```

Comment:
Incomplete text?

Reply:
i'm glad you commented on it though, it looks like i might have an extra space after the `(continue)` or maybe not. I'll check!